src/applications/combined-debt-portal/combined/components/ComboAlerts.jsx

```diff
@@ -50,14 +52,14 @@ ComboAlert.Zero = () => {
       <ul>
         <li>
           <strong>For benefit debts</strong>, call the Debt Management Center
-          (DMC) at <va-telephone contact="8008270648" />(
-          <va-telephone tty contact="711" />
+          (DMC) at <va-telephone contact="8008270648" uswds />(
```

Comment:
@OptionSelect , fyi, there's no v3 version of this component. So I don't think adding the `uswds` prop is necessary to `va-telephone`.